frontend/src/pages/BenchmarkGlobal/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from "react";
```

Comment:
We might be able to skip having an extra page for this, and rather just display the appropriate level of hierarchy.

Reply:
Sure, I can work on that 👍

Reply:
That sounds great!